backend/app/controllers/pub_sub_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PubSubController < FayeRails::Controller
```

Comment:
Is this really needed, there is only one row implementationf or each method? Shouldn't we just use that row?